src/main/java/net/atos/zac/app/admin/model/RESTZaakbeeindigReden.java

```diff
@@ -7,7 +7,7 @@
 
 public class RESTZaakbeeindigReden {
 
-    public Long id;
+    public String id;
```

Comment:
This change in contract, how does that affect the Frontend? Does that expect numbers now and will now have to be able to handle strings?

Reply:
Frontend works with string, that's why no change needed. I suspect they treat/expect the if as UUID